roles/run/files/schema.json

```diff
@@ -101,6 +110,43 @@
                                     "enum": [
                                         "module_openapi"
                                     ]
+                                },
```

Comment:
Can we also update README to reflect the changes and the new options that are being added?